src/service/addrVotes.ts

```diff
@@ -50,7 +50,7 @@ export class AddrVotes {
       tiebreaker = new Multiaddr(Object.keys(this.tallies)[0]);
     }
     const tiebreakerStr = tiebreaker.toString();
-    let best: [string, number] = [tiebreakerStr, this.tallies[tiebreakerStr]];
+    let best: [string, number] = [tiebreakerStr, this.tallies[tiebreakerStr] || 0];
```

Comment:
Should we use `??` instead? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Yes, good advice!